content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -122,314 +218,403 @@ Accounting.
+Integration, as detailed in the :ref:`italy/document-types` section.
 
-      - It is a **domestic** transaction (i.e., the partner is from Italy);
-      - The buyer's data is **insufficient** for a regular invoice;
-      - The **required fields** for a regular invoice (address, ZIP code, city, country) are
-        provided;
-      - The total amount VAT included is **less** than **400 EUR**.
+Be sure that you replace the :guilabel:`0% Sale Taxes` on the Vendor Bill you received with the ones
+you're supposed to pay to the :abbr:`AdE (Agenzia delle Entrate)`. A **button** then appears on the
+top of the single Vendor Bill form.
 
-      .. note::
-         The 400 EUR threshold was defined in `the decree of the 10th of May 2019 in the Gazzetta
-         Ufficiale <https://www.gazzettaufficiale.it/eli/id/2019/05/24/19A03271/sg>`_. We advise you
-         to check the current official value.
+.. figure:: italy/italy-edi-tax-integration-button.png
+   :align: center
+   :alt: EDI Send Tax Integration button
 
-   .. tab:: `TD24`
+   EDI Send Tax Integration button
 
-      Deferred invoices.
+When pressed, an **XML** file of the appropriate :guilabel:`Document Type` will be generated, attached
+to the Bill and then it will be sent following the same process as invoices, which is described here
+below.
 
-      The **deferred invoice** is an invoice that is **issued at a later time** than the sale of
-      goods or the provision of services. A **deferred invoice** has to be issued at the latest
-      within the **15th day** of the month following the delivery covered by the document.
+.. _italy/document-types:
 
-      It usually is a **summary invoice** containing a list of multiple sales of goods or services,
-      carried out in the month. The business is allowed to **group** the sales into **one invoice**,
-      generally issued at the **end of the month** for accounting purposes. Deferred invoices are
-      default for **wholesaler** having recurrent clients.
+Mastering the Document Types
+----------------------------
 
-      If the goods are transported by a **carrier**, every delivery has an associated **Documento di
-      Transporto (DDT)**, or **Transport Document**. The deferred invoice **must** indicate the
-      details of all the **DDTs** information for better tracing.
+The :abbr:`SdI (Sistema di Interscambio)` does not only require businesses to send customer invoices
+through the :abbr:`EDI (Electronic Data Interchange)` but there are also other documents that need
+sending.
 
-      .. note::
-         E-invoicing of deferred invoices requires the `l10n_it_stock_ddt`
-         :ref:`module <italy/modules>`. In this case, a dedicated :guilabel:`Tipo Documento` `TD24`
-         is used in the e-invoice.
+All the different configurations are technically identified by a :guilabel:`Document
+Type` code. We do **not** show the :guilabel:`Document Type` to the user, but it is used in the
+generation of the **XML** file.
```

Comment:
It's inside the XML file, customers know about that. We don't show it as a field in Odoo, but they have to know the values well, because they're held accountable for sending the right files with the correct Document Type.

They often open tickets saying "I have this invoice but the Document Type is not correct" even if it's not shown in Odoo. Same for our colleagues, many do not know why a Document Type is chosen over another.
This documentation should serve as a repository for that knowledge.

(Just as a random idea... could we consider showing it as readonly computed field, as we already somehow compute it? 🤔) @tsb-odoo @jco-odoo  ?

I don't know how to put that written into clear words.

Reply:
I don't see any valid reason to display a field with such technical information. It will then give the user the impression the issue is that the field shouldn't be readonly so that he would have control on it. But that's not the case. The thing is that our italian e-invoicing must work out of the box without the user having to ask himself about the document type. @lordkrandel @jco-odoo